.github/workflows/deploy-fleet-website.yml

```diff
@@ -46,6 +46,12 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
 
+    # Set the Go version
```

Comment:
"Install the right version of Go for the Golang child process that we are currently using for signing"